WordPress/Classes/ViewRelated/NUX/ContentViews/UnifiedPrologueEditorContentView.swift

```diff
@@ -0,0 +1,87 @@
+import UIKit
+import SwiftUI
+
+class UnifiedPrologueEditorContentView: UIView {
+
+    init() {
+        super.init(frame: .zero)
+        let controller = UIHostingController(rootView: UnifiedPrologueEditorView())
+        controller.view.translatesAutoresizingMaskIntoConstraints = false
+        controller.view.backgroundColor = .clear
+        addSubview(controller.view)
+        pinSubviewToAllEdges(controller.view)
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+}
+
+
+struct UnifiedPrologueEditorView: View {
+
+    var body: some View {
+
+        VStack {
+            Spacer()
+            ZStack {
+                RoundedRectangle(cornerRadius: 4, style: .continuous)
```

Comment:
This RoundedRectangle with corner radius and colour looks like a good candidate to pull out into a small separate view, as we use it in a couple of places and on other screens.